pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -455,3 +467,16 @@ func MakeRelative(path, base string) (string, error) {
 	}
 	return path, nil
 }
+
+// HomeDir returns the home directory for the current user
+func HomeDir() string {
```

Comment:
I don't like public utility methods like this - does it need to be public for a specific reason?  If so, it should be in `pkg/util/shell/home` or something.